ledger/notary-plugin/src/main/kotlin/net/corda/v5/ledger/notary/pluggable/PluggableNotaryClientFlowProvider.kt

```diff
@@ -0,0 +1,18 @@
+package net.corda.v5.ledger.notary.pluggable
+
+import net.corda.v5.ledger.common.Party
+import net.corda.v5.ledger.utxo.transaction.UtxoSignedTransaction
+
+/**
+ * This interface is used to instantiate the client-side logic (flow) of the plugin.
+ * This is needed because the plugins will be installed on node startup.
+ *
+ * Implementers of this interface must have the [PluggableNotaryType] annotation,
+ * otherwise an exception will be thrown on startup.
+ *
+ * The client-side of the plugin must have the same constructor parameters as the
+ * [create] function (notary, stx, context).
```

Comment:
I'm not sure where `context` comes in as this isn't actually in the `create` function signature, but isn't this comment redundant since a class must implement the `create` function as defined in the interface in order to compile?

Reply:
Yeah good point, will remove this comment altogether.